website/docs/docs/cloud/use-visual-editor.md

```diff
@@ -0,0 +1,80 @@
+  - The environment ID must have had runs that generated catalogs in it.
+
+- For example, if my region is North America multi-tenant, account ID is 10, environment ID with a generated catalog run is 100, my URL should be:
+
+  - `https://cloud.getdbt.com/visual-editor/10/env/100/`
+
+<Lightbox src="/img/docs/dbt-cloud/visual-editor/visual-editor.jpg" width="80%" title="Visual editor interface that contains a node toolbar and canvas." />
+
+## Create a model
+To create a new dbt SQL model, click on **Create a new model** and perform the following steps:
+
+1. Drag an operator from the operator toolbar and drop it onto the canvas.
+2. Click on the operator to open its configuration panel:
+	- **Table**: Select the table and columns you want to use.
+	- **Join**: Define the join conditions and choose columns from both tables.
+	- **Select**: Pick the columns you need from the table.
+	- **Aggregate**: Specify the aggregation functions and the columns they apply to.
+	- **Formula**: Add the formula to create a new column. Use the built-AI code generator to help generate SQL code by clicking on the question mark (?) icon. Enter your prompt and wait to see the results.
+	- **Filter**: Set the conditions to filter data.
+	- **Order**: Select the columns to sort by and the sort order.
+	- **Limit**: Set the maximum number of rows you want to return.
+3. View the **Output** and **SQL Code** tabs. 
+	- Each operator has an Output tab that allows you to preview the data from that configured node.
+	- The Code tab displays the SQL code generated by the node's configuration. Use this to see the SQL for your visual model config.
+3. Connect the operators by using the connector by dragging your cursor between the operator's "+" start point and linking it to the other operators you want to connect to. This should create a connector line. 
+	- Doing this allows the data to flow from the source table through various transformations you configured, to the final output.
+5. Keep building your dbt model and ensure you confirm the out through the **Output** tab.
+
+<!-- 
+### Configure nodes
+- Built-in AI code generator
+
+### View output
+-->
+
+## Edit an existing model
+To edit an existing model, navigate to the visual editor and click on the **Edit a model** button on the upper right. This will allow you to select the model you'd like to edit.
+
+<Lightbox src="/img/docs/dbt-cloud/visual-editor/edit-model.jpg" width="90%" title="Edit a model iin the visual editor using the 'Edit a model' button." />
+
+## Version control
+
+Testing and documenting your models is an important part of the development process.  
+
+Stay tuned! Coming very soon, you'll be able to version control your dbt modes in the visual editor. This ensures you can track changes and revert to previous versions if needed.
+
+## Limitations
```

Comment:
how to integrate with explorer and ide?

Reply:
no integrations yet, once we can save branch, they'll be able open the branch in ide.